docs/concepts/internationalization.md

```diff
@@ -37,6 +37,29 @@ By default, `#getDescriptionId` will return `block.` or `item.` prepended to the
 The only purpose of a translation key is internationalization. Do not use them for logic. Use registry names instead.
 :::
 
+Mod metadata
+------------
```

Comment:
Please use the new format (# and ##). Also, while you're at it, could you update the whole file to the new format?

Reply:
Done in a second commit :+1: